src/graphql/operations/activeSpaces.ts

```diff
@@ -0,0 +1,16 @@
+import { checkLimits, fetchActiveSpaces, handleRelatedSpaces, PublicError } from '../helpers';
+import log from '../../helpers/log';
+
+export default async function (_parent, args, _context, info) {
+  checkLimits(args, 'activeSpaces');
+  try {
+    const { spaces, metrics } = await fetchActiveSpaces(args);
```

Comment:
Can we move the function `fetchActiveSpaces` on this file? It's only being used here, it's better if we keep the logic in same place and use helpers for functions that are used on different files.